src/data/actions/CLEAR_GATE_CELLAR/getApy.ts

```diff
@@ -0,0 +1,4 @@
+export const getApy = () => {
+  // the value is pulled from cellarDataMap.overrideApy
+  return null
```

Comment:
If this is being overridden elsewhere, do we need this function? Couldn't we have some sort of optional chaining that makes a call to the common `getApy` if the value isn't being pulled in from the `cellarDataMap`?

Reply:
We don't really need this function. Yes We could. Good point, Thanks!